libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -102,12 +110,15 @@
         [style.maxHeight]="!mobile ? maxHeight : 'auto'"
         (focusEscapeList)="handleListFocusEscape($event)"
         [style.minWidth]="'100%'"
+        role="listbox"
```

Comment:
You prob need `aria-activedescendant` too. Please check: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

Reply:
Not required as per a11y team the list item count and position has to read with the value which is happening.